src/Proto3/Suite/Form/Encode/Core.hs

```diff
@@ -354,7 +355,7 @@ class Field name a message
     field :: forall names . a -> Prefix message names (Occupy message name names)
 
 instance forall (name :: Symbol)
-#if MIN_VERSION_ghc(9,4,0)
+#if MIN_VERSION_ghc_lib_parser(9,4,0)
```

Comment:
I'm afraid that this particular condition really **_is_** about the compiler version, not the library version.  GHC 9.2.8 does not allow this style of runtime representation polymorphism.  Perhaps reverting this one line would help CI succeed.

Reply:
Actually, since you're eliminating the dependence upon the `ghc` package, it might have to be a surrogate, like `MIN_VERSION_base(4,17,0)`.

Reply:
Thank you, your feedback helped me identify the issue right away.
After some research, I found that the `__GLASGOW_HASKELL__` macro can be used to detect the GHC version without depending on packages, so I'll use that approach instead.